safehttp/task_test.go

```diff
@@ -26,9 +27,16 @@ import (
 	"github.com/google/safehtml"
 )
 
-func TestResponseWriterSetCookie(t *testing.T) {
+func stubResponseWriter(dispatcher safehttp.Dispatcher, rw http.ResponseWriter) safehttp.ResponseWriter {
```

Comment:
The `safehttp.NewResponseWriter` API used to return an incomplete `ResponseWriter` (the `ResponseWriter` needs to call interceptors associated with the current handler - this was missing among other things).

We currently settle for a (still broken) implementation, but the next refactoring PR will address the testing issues we have here.